.github/workflows/build.yml

```diff
@@ -19,6 +22,10 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.ref }}
           repository: ${{ github.event.pull_request.head.repo.full_name }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.18.0
```

Comment:
do we want to go ahead and enforce node-version: 16 or wait for us to make that change in cdklabs-projen-project-types (my preference)?

Reply:
Some packages already fail so you might as well.

But note that there's a difference between enforcing it and stop testing for it.
Personally I'd rather do the latter and only enforce it in a few months or even only when we actually want to use certain language features.